src/script/main/app.ts

```diff
@@ -436,6 +432,7 @@ export class App {
       if (supportsMLS()) {
         //if mls is supported, we need to initialize the callbacks (they are used when decrypting messages)
         conversationRepository.initMLSConversationRecoveredListener();
+        registerMLSConversationVerificationStateHandler(this.updateConversationE2EIVerificationState);
```

Comment:
Moved handler initialiser to the point when all the conversations are already fetched from backend.

Reply:
I don't think moving this handler matters that much since it relies on events from the core to actually do anything.

Reply:
Don't we need to have the conversation in local state to be able to search it by group id? Now we're sure it will be there, there's no harm i believe

Reply:
It's always good not to subscribe to something if you know you don't need it. It's good to have this `if` (in case that's really what is being debated here)